src/test/java/seedu/pennywise/logic/parser/AddCommandParserTest.java

```diff
@@ -164,16 +181,6 @@ public void parse_compulsoryFieldMissing_failure() {
                 TYPE_INCOME + DESC_LUNCH + VALID_AMT_LUNCH + DATE_LUNCH + TAG_LUNCH,
                 expectedMessage);
 
-        // missing date prefix
-        assertParseFailure(
-                parser,
-                TYPE_EXPENDITURE + DESC_LUNCH + AMT_LUNCH + VALID_DATE_LUNCH + TAG_LUNCH,
-                expectedMessage);
-        assertParseFailure(
-                parser,
-                TYPE_INCOME + DESC_LUNCH + AMT_LUNCH + VALID_DATE_LUNCH + TAG_LUNCH,
-                expectedMessage);
-
```

Comment:
This was removed since the date prefix missing should still lead to a successful parse.